### ISSUE TITLE
[flutter_midi] Lag

### ISSUE THREAD

**Issue by lastant** [AppleEducate/plugins#63]
There's a significant delay on my Android between pressing a button and hearing the sound. How can I try to make it faster?

**Comment by lastant** [comment]
I guess this is a problem with the underlying library..

**Comment by TheKashe** [comment]
Yeah, it's the underlying library, which has seen no updates for many years.

There's [this](https://github.com/billthefarmer/mididriver)  alternative, but [it doesn't seem to have an option to load custom sf2 or dsl](https://stackoverflow.com/questions/5125911/how-to-load-custom-dls-or-soundfonts-to-android-midi?lq=1). 

Then there's the [fluidsynth](https://github.com/VolcanoMobile/fluidsynth-android) which seems awesome, but this android port has no documentation or java wrapper.

**Comment by lastant** [comment]
I managed to compile Fluidsynth and make it work using the wrapper from jOrgan project. But then I found an even better library BASS (http://www.un4seen.com/) that is already pre-compiled for multiple platforms, has a wrapper, great documentation and excellent performance.

**Comment by rodydavis** [comment]
Wow awesome! Do you have an example? @lastant
